<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtmrKeyGeneratorClass</class>
 <widget class="QDialog" name="VtmrKeyGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VtmrKeyGenerator</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>机器码</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="LEMachinecCde"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>注册码</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="LERegistCode">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>有效期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBYear">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>年</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBMonth">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>月</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBDay">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>日</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnGenRegistCode">
       <property name="text">
        <string>生成注册码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopyRegistCode">
       <property name="text">
        <string>拷贝注册码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VtmrKeyGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
